lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -329,7 +329,33 @@ class ObjCPrinter : private DeclVisitor<ObjCPrinter>,
     os << "@end\n";
   }
 
+  bool isEmptyExtensionDecl(ExtensionDecl *ED) {
+    auto members = ED->getMembers();
+    SmallVector<Decl *, 4> includedMembers;
+    std::copy_if(members.begin(), members.end(),
```

Comment:
You can do even better by using [`std::any_of`](http://en.cppreference.com/w/cpp/algorithm/all_any_none_of).